test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -10007,5 +10007,24 @@ TEST_F(AtenXlaTensorTest, TestEmbeddingBackward) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestEarlySyncLiveTensors) {
+  torch::Tensor scalar_tensor =
+      torch::scalar_tensor(1., torch::TensorOptions(torch::kFloat));
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor xla_scalar_tensor = CopyToDevice(scalar_tensor, device);
+    torch::Scalar scalar = xla_scalar_tensor.item();
```

Comment:
Can you add a value check here, like
```
torch::Scalar scalar = scalar_tensor.item();
ForEachDevice([&](const torch::Device& device) {
.....
torch::Scalar xla_scalar = xla_scalar_tensor.item();
EXPECT_TRUE(scalar == xla_scalar); // I think scalar supports == operator
}
```
we still want to make sure that `xla local_scalar_dense` give the same value as the native `local_scalar_dense`